docs/articles/2023/20231012-normalized_layer_precalc_stats.md

```diff
@@ -0,0 +1,225 @@
+    iterator = query.X("normalized").tables()
+    
+    # Iterate over the normalized matrix.
+    # Get an iterative slice as pyarrow.Table
+    raw_slice = next (iterator)
+    
+    # Perform analysis
+    
+    # close the query
+    query.close()
+```
+
+And the equivalent code in R.
+
+```r
+library("cellxgene.census")
+library("tiledbsoma")
+
+human <-  census$get("census_data")$get("homo_sapiens")
+query <-  human$axis_query(
+  measurement_name = "RNA",
+  obs_query = SOMAAxisQuery$new(
+    value_filter = "tissue == 'brain' & sex == 'male'"
+  )
+)
+
+# Set iterable for normalized matrix
+iterator <-  query$X("normalized")$tables()
+
+# Iterate over the normalized matrix.
+# Get an iterative slice as an Arrow Table
+raw_slice <-  iterator$read_next()
+
+# Perform analysis
+```
+
+### Utilizing pre-calculated stats for querying `obs` and `var`
+
+To filter cells or genes based on pre-calculated statistics and export to AnnData, you can use the new metadata variables as value filters.
+
+For example you can you can add a filter to query cells with more than 500 genes expressed in addition to other filters. In Python this looks like the following.
+
+```python
+import cellxgene_census
+
+with cellxgene_census.open_soma(census_version = "latest") as census:
+    adata = cellxgene_census.get_anndata(
+        census = census,
+        organism = "Homo sapiens",
+        obs_value_filter = "nnz > 500 and cell_type == 'sympathetic neuron'",
+        column_names = {"obs": ["tissue", "cell_type"]},
+        var_value_filter = "feature_id in ['ENSG00000161798', 'ENSG00000188229']",
+    )
+
+    print(adata)
+```
+
+In R, the equivalent code looks as follows.
+
+```r
+#Seurat
+library("cellxgene.census")
+library("Seurat")
+
+census <- open_soma(census_version = "latest")
+
+organism <- "Homo sapiens"
+gene_filter <- "feature_id %in% c('ENSG00000107317', 'ENSG00000106034')"
+cell_filter <-  "nnz > 500 & cell_type == 'sympathetic neuron'"
+cell_columns <- c("tissue", "cell_type")
+
+seurat_obj <- get_seurat(
+   census = census,
+   organism = organism,
+   var_value_filter = gene_filter,
+   obs_value_filter = cell_filter,
+   obs_column_names = cell_columns
+)
+
+#Single Cell Experiment
+
+library("SingleCellExperiment")
+
+sce_obj <- get_single_cell_experiment(
+   census = census,
+   organism = organism,
+   var_value_filter = gene_filter,
+   obs_value_filter = cell_filter,
+   obs_column_names = cell_columns
+)
+```
+
+## Help us improve these data additions
+
+We encourage you to engage with these new features in the Census API and share your feedback. This input is invaluable for the ongoing enhancement of the Census project.
+
+For further information on the new library-size normalized layer, please reach out to us at [soma@chanzuckerberg.com](soma@chanzuckerberg.com). To report issues or for additional feedback, refer to our [Census GitHub repository](https://github.com/chanzuckerberg/cellxgene-census/issues).
```

Comment:
what about "further information" on the other features?   Suggest:   "For further information on any new feature, please reach out ...."